packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.controller.js

```diff
@@ -55,9 +52,9 @@ export default class TrafficController {
         this.displayWarning = true;
       }
     }
-    if (this.ip) {
-      this.selectedIp = this.ip;
-      this.checkSelectedSubnet(this.ip);
+    if (this.ip()) {
+      this.selectedIp = this.ip();
+      this.checkSelectedSubnet(this.ip());
```

Comment:
```suggestion
    if (this.ip) {
      this.selectedIp = this.ip;
      this.checkSelectedSubnet(this.ip);
```

Reply:
the resolve method return a function so I need to do `this.ip()` to have the correct value for IP, same thing for dateTime